discover/deployment/articles/06-upgrading-to-liferay-7-1/02-preparing-for-upgrade-to-liferay-7-1.markdown

```diff
@@ -23,6 +23,12 @@ a summary of the preparation steps:
 
 This tutorial describes these steps in detail. 
 
+**Important:** If you're running DXP 7.0 Fix Pack 51 or earlier, upgrade to the 
+latest DXP 7.0 fix pack. This assures that all 7.0 upgrade processes run prior
+to upgrading to 7.1. See 
+[LPS-83449](https://issues.liferay.com/browse/LPS-83449)
+for details. 
+
```

Comment:
Hi @jhinkey,

This is not correct because installing the fix pack 52 or higher once you already are in 7.0 won't run pending Core upgrade processes (in 7.1 you have the possibility to do it but not in 7.0). The only recomendation we can give is to install the latest available fix pack before executing the upgrade to DXP 7.0.

Let me know yout thoughs.

Thanks.

Reply:
Hi @achaparro ,

OK. So the 6.2 sysadmin should install the latest 6.2 fix pack before upgrading to 7.0?

Are there any instructions or things I should mention for the 7.0 sysadmin?

Thanks!

Reply:
Hi @jhinkey,

What I mean is that before executing any kind of upgrade they should install latest available fix pack in the target version. For example, if the upgrade from 6.2 to DXP 7.0 they should install the latest 7.0 fix pack before the upgrade, if they upgrade 7.1 they should install the latest 7.1 fix pack.

In case of the community users, they should install the latest available GA.

Thanks.